snake_game_high_score/scoreboard.py

```diff
@@ -0,0 +1,35 @@
+from turtle import Turtle
+ALIGNMENT = "center"
+FONT = ("Courier", 24, "normal")
+
+
+class Scoreboard(Turtle):
+
+    def __init__(self):
+        super().__init__()
+        self.score = 0
+        with open("data.txt") as data:
```

Comment:
You don't have this file in the repo -- I figured out that it was a text file named `data.txt` with a single number in it form context (logically it should probably have a 0 in it, since you wouldn't have a high score before the first time you play)